src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -654,7 +654,8 @@ private void PulseMakerRunStarted()
         /// <param name="markNodesAsDirty">Set this parameter to true to force 
         ///     reset of the execution substrait. Note that setting this parameter 
         ///     to true will have a negative performance impact.</param>
-        internal void ResetEngine(EngineController controller, bool markNodesAsDirty = false)
+        /// <param name="forceAutomaticWithoutRun">This parameter will indicate not execute the Graph but the RunMode will be Automatic</param>
```

Comment:
Maybe this comment can be improved to something like:
`This parameter, if true, will prohibit the graph from executing when RunMode is set to Automatic`

Reply:
@zeusongit the comment was updated in several places, check the next commit:
https://github.com/DynamoDS/Dynamo/pull/12916/commits/6e3af78f9f27e36771c3ceb54e8ea240bf3c9047